task_lib.c

```diff
@@ -78,7 +80,7 @@
 // than the adaptive pthread mutexes, but performance is terrible if the worker threads ever get
 // context switched by the Linux scheduler. It is slightly easier however to gather profiling
 // statistics about the time spent in TFD locks with gprof using ticketlocks
-#define USE_TICKET_LOCKS
+// #define USE_TICKET_LOCKS
```

Comment:
As per comment above.  Please merge.  We're using ticket locks for now as a baic matter of practise.